SPECS/rpm/rpm.spec

```diff
@@ -107,31 +107,38 @@ Python3 rpm.
 %autosetup -n rpm-%{name}-%{version}-release -p1
 
 %build
-sed -i '/define _GNU_SOURCE/a #include "../config.h"' tools/sepdebugcrcfix.c
 # pass -L opts to gcc as well to prioritize it over standard libs
 sed -i 's/-Wl,-L//g' python/setup.py.in
 sed -i '/library_dirs/d' python/setup.py.in
 sed -i 's/extra_link_args/library_dirs/g' python/setup.py.in
 
 ./autogen.sh --noconfigure
+#Enables new database format
 %configure \
     CPPFLAGS='-I/usr/include/nspr -I/usr/include/nss -DLUA_COMPAT_APIINTCASTS' \
-        --program-prefix= \
-        --disable-dependency-tracking \
-        --disable-static \
-        --with-vendor=mariner \
-        --enable-python \
-        --with-cap \
-        --with-lua \
-        --disable-silent-rules \
-        --with-external-db \
-        --with-selinux
+    --program-prefix= \
+    --with-crypto=openssl \
+    --enable-ndb \
+    --disable-dependency-tracking \
+    --disable-static \
+    --with-vendor=mariner \
+    --enable-python \
+    --with-cap \
+    --disable-silent-rules \
+    --with-selinux
+
+# Remove manpages translations
+rm -r docs/man/{fr,ja,ko,pl,ru,sk}
+
 make %{?_smp_mflags}
 
 pushd python
 python3 setup.py build
 popd
 
+# Set provided python versions
+sed -i 's/@MAJORVER-PROVIDES-VERSIONS@/2.7,3.6/' %{SOURCE3}
```

Comment:
Let's use the `%python2_version` and `%python3_version` macros instead, so that they are not hard-coded here.